lib/libutee/include/apdu_pta.h

```diff
@@ -0,0 +1,30 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+/*
+ * Copyright (C) 2021, Foundries Limited
+ */
+
+#ifndef __APDU_PTA_H
+#define __APDU_PTA_H
```

Comment:
again: almost all PTAs in optee_os use _pta__\<_some_naming_>.h_ as header file name (but scp03 PTA (cb1b1ecd336abb5c3cca92eb702f2a794b023e40) and rng PTA (b2ac1d0fa34ca28382fcedb30d938ccd4e8a5316). Please, help for the consistency in optee_os, and use standard _pta_apdu.h_.

@jforissier, @jenswi-linaro @ruchi393, once for all for my information: do you think it's worth having some consistency in optee_os PTAs header files naming/layout/API description format? There are the reference documentation for PTA embedded in OP-TEE core. If you don't think so, fine, I won't ask the question again.

Reply:
I agree. When I submitted those drivers I was asked to have the name inverted so I remember changing them. Now you are asking for the opposite (with an emphasis on 'again'). So yeah, while I personally don't care too much, having some consistency would be great. what is going to be then?

Reply:
I'll push again as it is. I can do another commit to rename these three header files - I probably should but please confirm.